addons/calendar/models/calendar.py

```diff
@@ -159,7 +159,7 @@ def create(self, vals_list):
     def copy(self, default=None):
         raise UserError(_('You cannot duplicate a calendar attendee.'))
 
-    def _send_mail_to_attendees(self, template_xmlid, force_send=False, force_event_id=None):
+    def _send_mail_to_attendees(self, template_xmlid=None, template_id=False, force_send=False, force_event_id=None):
```

Comment:
template_id and template_xmlid are a bit 'double emploi'. Better to keep template_id and remove template_xmlid.
Caller should get template from xmlid if needed.
And we can take the opportunity to rename template_id in template to 'ask' the caller to give a template and not an id to the method.